extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1010,9 +1017,10 @@ export class AmpA4A extends AMP.BaseElement {
   /**
    * Gets the Ad URL to send an XHR Request to.  To be implemented
    * by network.
+   * @param {Array=} opt_rtcResponsesPromise
```

Comment:
Is this an array?  I thought it was object whose key is the vendor name or domain of the custom url to result as JSONObject

Reply:
Let's make a typedef for the items in this array.  I believe it is:
{rtcResponse: ?Object<string, *>, rtcTime: number, callout: string} where callout could be url or vendor?

Reply:
done